Live stream improvements

fantapop opened the pull request.

Commit:
Fix json syntactical issue

Commit:
Database should be multi az

Commit:
Allow db access from bastion host

Commit:
infra/scripts to support live-video

Including:

top level cloudformation for "secondary" datacenters
scripts for initializing the environments
wowza security group

fantapop commented:
Note: some of the commits here were sitting on my box for the last 6 months.  I went ahead and added them here.  The new stuff is all in "infra/scripts to support live-video"

brian-traceme commented:
PR #1 !!!!

brian-traceme commented on conf/prod.yaml:
```diff
@@ -6,8 +6,7 @@
     { "ParameterKey": "ProductServiceCount", "ParameterValue": "2" },
     { "ParameterKey": "DatabaseAdminUsername", "ParameterValue": "tracemeprodroot" },
-    { "ParameterKey": "DatabaseAdminPassword", "ParameterValue": "d4da6Uf@drUHucEbRuspe7$uVad7XA
-      " },
```
What? the password doesn't have a newline? that would be super secure!

brian-traceme commented on stacks/database.yaml:
```diff
@@ -76,8 +76,10 @@ Resources:
+
       Engine: postgres
       EngineVersion: 9.6
+      MultiAZ: true
```
multiple arizona?

Commit:
Add stage to tags on components

This is used to be able to tell them apart when selecting subnets to use
when spinning up live video servers

Commit:
remove some trailing spaces

Commit:
better ignore vim swap files
